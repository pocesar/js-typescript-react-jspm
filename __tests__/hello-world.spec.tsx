
import React from 'react'
import renderer from 'react-test-renderer'
import HelloWorld from '../lib/components/hello-world'

const HWAN: any = HelloWorld

describe('<HelloWorld />', () => {
  test('matches snapshot', async () => {
    const simple = renderer.create(
      <HWAN />
    )

    expect(simple.toJSON()).toMatchSnapshot()
  })
})
