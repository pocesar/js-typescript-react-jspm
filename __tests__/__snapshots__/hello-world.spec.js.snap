// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld /> matches snapshot 1`] = `
<div>
  <label>
    Hello World! 
  </label>
  <button
    onClick={[Function]}
  >
    Add
  </button>
  <button
    onClick={[Function]}
  >
    Sub
  </button>
  <button
    onClick={[Function]}
  >
    Hi
  </button>
</div>
`;
